### PR TITLE
More one degree simulation

### PR DESCRIPTION
**Pull request by simone-silvestri**
this PR is to continue working and collaborating on the `experiments/one_degree_simulations/one_degree_simulation.jl` file to get a stable one degree simulation set up

### CODE REVIEW

**Comment by simone-silvestri** [comment]
The one degree simulation as it stands NaNs with velocities increasing drastically very quickly.

File: src/DataWrangling/inpaint_mask.jl [hunk 1]
```diff
@@ -140,7 +140,7 @@ Arguments
 - `inpainting`: The inpainting algorithm to use. For the moment, the only option is `NearestNeighborInpainting(maxiter)`, 
                 where an average of the valid surrounding values is used `maxiter` times.
 """
-function inpaint_mask!(field, mask; inpainting=NearestNeighborInpainting(10))
```
Comment (simone-silvestri) [inline src/DataWrangling/inpaint_mask.jl:143]: found the mistery of the crashing

Comment (glwagner) [reply src/DataWrangling/inpaint_mask.jl:143]: damn the maps

**Comment by glwagner** [comment]
should we merge this asap

File: experiments/one_degree_simulation/one_degree_simulation.jl [hunk 2]
```diff
@@ -74,7 +73,7 @@ forcing = (T=FT, S=FS)
 ##### 
 
 momentum_advection = VectorInvariant()
-tracer_advection   = Centered(order=2)
+tracer_advection   = WENO()
```
Comment (glwagner) [inline experiments/one_degree_simulation/one_degree_simulation.jl:76]: you sure?

File: src/DataWrangling/inpaint_mask.jl (hunk 1 above)
Comment (glwagner) [reply src/DataWrangling/inpaint_mask.jl:143]: i think we should merge this and then open a new PR from the same brnach what do you think?

File: experiments/one_degree_simulation/one_degree_simulation.jl (hunk 2 above)
Comment (simone-silvestri) [reply experiments/one_degree_simulation/one_degree_simulation.jl:76]: Nono I reverted to centered. I was trying to see if centered was the issue

**Comment by simone-silvestri** [comment]
there is a problem with GM, see https://github.com/CliMA/Oceananigans.jl/issues/3938. However probably better to merge this and continue in another PR

**Review by glwagner — APPROVED** [review]

**Comment by navidcy** [comment]
Should we include the 1-degree, tripolar simulation in the docs?

**Comment by glwagner** [comment]
not until we sort out fluxes...

**Pull request merged**